tdrs-backend/tdpservice/parsers/validators/category3.py

```diff
@@ -299,13 +299,13 @@ def validate(record, row_schema):
         )
 
         if FAMILY_AFFILIATION == 2 and (
-            CITIZENSHIP_STATUS == 1 or CITIZENSHIP_STATUS == 2
+            CITIZENSHIP_STATUS == 1 or CITIZENSHIP_STATUS == 2 or CITIZENSHIP_STATUS == 3
```

Comment:
@klinkoberstar @andrew-jameson are we sure this ticket is still needed? I've tested these ACs in staging today (2/7) and each is already satisfied. 

It also appears that the function here is related to cat 3 check that considers family affiliation, citizenship, and SSN. SSN isn't referenced in the ACs. 

I believe there are a few citizenship-related tickets so it's very possible I may be missing something. Test files below:



[2024.Q1.Closed Case Data.txt](https://github.com/user-attachments/files/18709350/2024.Q1.Closed.Case.Data.txt)
[2024.Q1.Active Case Data.txt](https://github.com/user-attachments/files/18709357/2024.Q1.Active.Case.Data.txt)